roles/capsule_certs_generate/molecule/satellite/prepare.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Prepare
+  hosts: all
+  gather_facts: true
+  become: true
+  vars:
+    foreman_repositories_version: "2.4"
+    foreman_repositories_katello_version: "4.0"
+  roles:
+    - theforeman.operations.foreman_repositories
+    - theforeman.operations.puppet_repositories
+    - fake_installer_rpm
+  post_tasks:
```

Comment:
this could've been a normal role invocation, there is no need for `include_role`, right?

Reply:
Nooo.... yes.